src/test/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionSpec.scala

```diff
@@ -290,42 +292,15 @@ class SubmissionSpec(_system: ActorSystem) extends TestKit(_system) with FlatSpe
       status
     }
 
-    // the mock exec service always returns 2 successes and a failure and there should be 2 batches
-    assert( newSubmissionReport.notstarted.size == 0 ) // this not started count includes only those with bad inputs, not errors
-    assert( newSubmissionReport.workflows.size == 4 )
+    assert( newSubmissionReport.notstarted.size == 0 )
+    assert( newSubmissionReport.workflows.size == 6 )
 
     checkSubmissionStatus(workspaceService, newSubmissionReport.submissionId)
 
     val submission = runAndWait(submissionQuery.loadSubmission(UUID.fromString(newSubmissionReport.submissionId))).get
-    assert( submission.workflows.size == 4 )
-    assert( submission.notstarted.size == 2 ) // this not starated count includes errors
+    assert( submission.workflows.forall(_.status == WorkflowStatuses.Queued) )
   }
 
-  it should "match workflows to entities they run on in the right order" in withWorkspaceServiceMockExecution { execService => { workspaceService =>
```

Comment:
I didn't see your comment when I wrote this, so this ^ was unintentional sass.

I prefer my sass to be intentional, for the record.